demo.html

```diff
@@ -5,11 +5,41 @@
     <title>th-data-split Demo</title>
     <script src="../webcomponentsjs/webcomponents.js"></script>
     <link rel="import" href="th-data-split.html">
+
   </head>
   <body unresolved>
 
-    <p>An example of `th-data-split` looks like this:</p>
-    <th-data-split></th-data-split>
+    <template id="auto-bind-demo" is="auto-binding">
+      <p>Input:</p>
+      <pre id="input"></pre>
+      <script>
```

Comment:
so moving the script tag inside auto binding template resolved the problem of element being inaccessible?